src/blocks/pricing-table/index.jsx

```diff
@@ -1,16 +1,13 @@
 // WordPress dependencies.
```

Comment:
This doesn't seem to work correctly with memberships.
Editing a membership the correct pricing table is displayed, although in the Inspector I still see Course/Select course. While I should see Membership/Inherit from current. Right?
![screenshot-20230807_104825_733504507](https://github.com/gocodebox/lifterlms/assets/7689242/33ff668f-1443-42f6-a8c4-88d3747d50c6)

Also it should say "Select Membership" not "Select Course" when a membership is selected.

Reply:
You seem to have missed this too

Reply:
@eri-trabiccolo this should be working correctly now